charts/keess/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+1. Keess deployed successfully.
```

Comment:
What's the purpose of this file?

Reply:
This is the message displayed after the char installation.